CHANGELOG.md

```diff
@@ -4,11 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [2.2.3] - unreleased
+## [2.2.4] - 2021-06-04
+- Minor dependency version updates
+
+## [2.2.3] - 2021-05-21
 ## Changed
 - updated `singlePlay` response to not return an object containing the boolean, but just the boolean itself.
 
-## [2.2.2] - 2021-05-03
+## [2.2.2] - 2021-05-05
```

Comment:
Not sure why this date was updated?

Reply:
Been a few months. But I believe the release was delayed (originally it was going to be May 3) and the 05-05 date is correct. But that update never got made in develop